src/kOS/Logger.cs

```diff
@@ -44,11 +44,8 @@ public virtual void Log(Exception e)
 
                 message += "\n" + LINE_RULE + "           VERBOSE DESCRIPTION\n";
                 
-                message += ((KOSException)e).VerboseMessage;
+                message += ((KOSException)e).VerboseMessage + "\n";
                 
-                // Fallback if there was no verbose message defined:
-                if (message == String.Empty)
-                    message += e.Message;
```

Comment:
This condition was literally impossible.  `message` was already set to `e.Message`, so the only way it is empty is if `e.Message` is empty, which doesn't do us any good at all.